cmd/agent/dist/conf.d/kubernetes_apiserver.d/conf.yaml.example

```diff
@@ -12,8 +12,36 @@ instances:
     #   - <KEY_1>:<VALUE_1>
     #   - <KEY_2>:<VALUE_2>
 
+    ## @param unbundle_events - boolean - optional - default: false
+    ## Makes each collected Kubernetes event be transformed into its own
+    ## Datadog event, instead of grouping them together by InvolvedObject.
+    #
+    # unbundle_events: true
+
+    ## @param collected_event_types - map of array of strings - optional
+    ## Specify which events to be collected. Map of Kind as key, array of event
+    ## reasons as value. Keys are case-insensitive, values are case-sensitive.
+    ## Only effective is unbundle_events is true, otherwise filtered_event_types
```

Comment:
Does "Only effective is unbundle_events is true" mean it's only effective when unbundle_events is true? If so, I'd suggest rewording to:
"Only effective when unbundle_events is true..."